tests/IntRangeTest.php

```diff
@@ -704,6 +704,18 @@ function doAnalysis(): void
                     '$_arr===' => 'non-empty-array<int<0, max>, int<0, max>>',
                 ],
             ],
+            'noErrorPushingBigShapeIntoConstant' => [
+                'code' => '<?php
+                        class DocComment
+                            {
+                                private const PSALM_ANNOTATIONS = [
+                                    "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "",
+                                    "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", ""
+                                ];
+                            }',
+                'assertions' => [
+                ],
+            ],
```

Comment:
Does this test actually fail before the fix? I tried the same thing with the actual values from DocComment but for some reason it passed fine.

Reply:
it does: https://psalm.dev/r/1529295950

Reply:
Although I noticed that it started failing after 52 values in there. DocComment must have around 55, so maybe you didn't include them all?

Reply:
@orklah Oh I see what happened, I used `--filter` for my test, but I forgot `paratest` doesn't support that, you have to use `phpunit` directly. I saw the success message and didn't notice it said 0 tests...